src/main/java/uk/gov/hmcts/reform/sscs/service/DocumentPlaceholderService.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.sscs.service;
+
+import static uk.gov.hmcts.reform.sscs.config.PlaceholderConstants.*;
+
+import java.util.HashMap;
+import java.util.Map;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.ccd.domain.RegionalProcessingCenter;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+
+@Service
+@Slf4j
+public class DocumentPlaceholderService {
+
+    public Map<String, Object> generatePlaceholders(SscsCaseData caseData) {
+        Map<String, Object> placeholders = new HashMap<>();
+
+        placeholders.put(CASE_CREATED_DATE_LITERAL, caseData.getCaseCreated());
+
+        setRegionalProcessingOfficeAddress(placeholders, caseData);
+        return placeholders;
+    }
+
+    private Map<String, Object> setRegionalProcessingOfficeAddress(Map<String, Object> placeholders, SscsCaseData caseData) {
+        RegionalProcessingCenter rpc;
+
+        if (hasRegionalProcessingCenter(caseData)) {
+            rpc = caseData.getRegionalProcessingCenter();
+        } else {
+            //FIXME: somehow add to some exception queue - this should be covered by another ticket
+            return null;
```

Comment:
Although this is for another ticket, presumably this is the same as `rpc` equal to `null`.

Reply:
yes i think so